examples/swarmion-with-next/.github/workflows/merge-main.yml

```diff
@@ -3,10 +3,17 @@ on:
     branches:
       - main
 
-concurrency: deploy_main
-
```

Comment:
Why remove this?
This prevents concurrent cloudformation deployments, I think we should keep it.

Maybe add a comment if it is not explicit enough

Reply:
This is a good catch, it had been removed from the other examples and I copy-pasted 😆